syscalls_other.go

```diff
@@ -0,0 +1,19 @@
+// +build !linux
+
+package water
+
+const (
```

Comment:
Just a thought -- Might be worth factoring out the const and type declarations into a separate source file, so that only platform-specific code (i.e., `createInterface`) appears in this file?  That way, if others update `ifReq` or amend the constants, only one source file need be updated, instead of one per platform.